rasgoql/requirements.txt

```diff
@@ -2,4 +2,5 @@ jinja2
 pandas
 python-dotenv
 pyyaml
-rasgotransforms~=1.0
+rasgotransforms>=1.0
```

Comment:
1. `rasgotransforms~=1.0` may be safer, let's rap battle about it.
2. `requests`: guessing this was for testing, do we actually need for pg support?